jitbuilder/build/rules/common.mk

```diff
@@ -51,6 +51,9 @@ $(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/il:
 $(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/ilgen:
 	mkdir -p $(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/ilgen
 
+$(RELEASE_INCLUDE)/$(JIT_OMR_DIRTY_DIR)/infra:
```

Comment:
I'm not super happy with this addition, especially since it's only needed because we *also* annotated the DataType class as OMR_EXTENSIBLE as part of #1492 (I think, these stacked PRs are really confusing).

But I don't really see a good way around it, since the OMR_EXTENSIBLE annotation is absolutely valuable.